.changeset/pre.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I believe this is all that is required to force it to publish with a beta tag instead of to the main `npm` package, but I find this whole changesets business a little mysterious.